docs/resources/gateway_partial.md

```diff
@@ -0,0 +1,197 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "konnect_gateway_partial Resource - terraform-provider-konnect"
+subcategory: ""
+description: |-
+  GatewayPartial Resource
+---
+
+# konnect_gateway_partial (Resource)
+
+GatewayPartial Resource
+
+## Example Usage
+
+```terraform
+resource "konnect_gateway_partial" "my_gatewaypartial" {
+  control_plane_id = "9524ec7d-36d9-465d-a8c5-83a3c9390458"
+  redis_ce = {
+    config = {
+      database    = 5
+      host        = "...my_host..."
+      password    = "...my_password..."
+      port        = 1
+      server_name = "...my_server_name..."
+      ssl         = true
+      ssl_verify  = true
+      timeout     = 10
+      username    = "...my_username..."
+    }
+    created_at = 10
+    id         = "...my_id..."
+    name       = "...my_name..."
+    tags = [
+      "..."
+    ]
+    type       = "redis-ce"
```

Comment:
Should `type` be a `const` in the spec? (applied via overlay)

The `redis_ce` nesting gives us this information

Reply:
@mheap could you please elaborate on this?
`The redis_ce nesting gives us this information`

Let me try making type a const meanwhile.

Reply:
There is only one valid value for `type` when using the `redis_ce` nesting in the provider. Instead of making the user provide it, we can send it by default using `const`

Reply:
Updated this PR, and raised one on platform api: https://github.com/Kong/platform-api/pull/1300